docs/guides/python/serverless-ai-api.mdx

```diff
@@ -0,0 +1,348 @@
+- _Portability:_ Nitric lets you **deploy anywhere** - AWS, GCP, Azure - by simply switching a setting, with all cloud-specific details handled for you. We'll deploy to AWS in this tutorial, but you could choose another cloud with minimal changes.
+- _Minimal configuration:_ The framework uses an “infrastructure from code” approach. We won't write separate infrastructure files; our Python code (and a tiny project config) is enough for Nitric to provision what's needed.
+
+Now, let's get our tools ready and dive in.
+
+## Prerequisites
+
+Before we begin, make sure you have the following:
+
+- **Python 3.11+** installed on your system (the latest 3.x is fine). Basic familiarity with Python is assumed.
+- **Nitric CLI** installed - this is the command-line tool to create, run, and deploy Nitric projects. You can install it by following the [installation instructions](/get-started/installation).
+- **uv** or **Pipenv** for managing Python dependencies. The Nitric Python templates use either uv or Pipenv.
+- An **OpenAI API key** if you want to integrate with OpenAI's GPT for summarization.
+- An **AWS account** (or Google Cloud or Azure account) with credentials configured on your machine, if you plan to deploy to the cloud. Nitric will use these credentials to provision resources. (For AWS, having the AWS CLI configured or environment vars set for access keys is enough.)
+
+<Note>
+  You can deploy to any of the three major clouds - Nitric supports all. In this
+  tutorial we'll mention AWS for concreteness, but feel free to use your
+  preferred cloud.
+</Note>
+
+## Project Setup - Your First Nitric App
+
+Nitric projects have a simple structure: your code (services, functions, etc.) lives in a directory (by default called `services/`), and a `nitric.yaml` config file describes high-level settings (like which files are services and what runtime to use). We'll use the Nitric CLI to scaffold a new project:
+
+1. **Create a new Nitric project:** Open a terminal and run:
+
+```bash
+nitric new my-summarizer py-starter
+```
+
+This will create a new folder `my-summarizer` with a Python starter template. (You can choose a different name for your project; `py-starter` selects the Python starter template.) If you omit the template, the CLI will interactively ask for a language/template. The CLI's `new` command guides you through project creation and offers ready-made templates.
+
+If none of the templates suit your needs, you can always create your own.
+
+2. **Install dependencies:** Navigate into the project and install the requirements. The template uses Pipenv, so you can do:
+
+```bash
+cd my-summarizer
+uv sync
+```
+
+This will install Nitric's Python SDK (and any other deps). If you used a different template, change this step to install dependencies accordingly.
```

Comment:
```suggestion
This will install Nitric's Python SDK (and any other dependencies). If you used a different template, change this step to install dependencies accordingly.
```